core/src/com/unciv/ui/overviewscreen/ReligionOverviewTable.kt

```diff
@@ -0,0 +1,106 @@
+package com.unciv.ui.overviewscreen
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.ui.Button
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.unciv.logic.civilization.CivilizationInfo
+import com.unciv.models.Religion
+import com.unciv.models.ruleset.Belief
+import com.unciv.models.translations.tr
+import com.unciv.ui.utils.*
+import kotlin.math.min
+
+class ReligionOverviewTable(
+    private val viewingPlayer: CivilizationInfo,
+    private val overviewScreen: EmpireOverviewScreen
```

Comment:
`@Suppress("UNUSED_PARAMETER")       // Keep all OverviewScreen Pages compatible` if you want - 'Problems' page of Studio immediately clear. Copied from resource sibling.